docs/REST API/rest-api.md

```diff
@@ -978,6 +978,10 @@ None.
       "enable_vulnerabilities_webhook":true,
       "destination_url": "https://server.com",
       "host_batch_size": 1000
+    },
+    "activities_webhook":{
+      "enable_activities_webhook":true,
+      "destination_url": "https://server.com"
```

Comment:
@getvictor will there be any cases where the parent object or either of these fields is `null` or not present?

Reply:
When reading configs, none of these fields will be null. We should follow the pattern with other webhook settings (`host_status_webhook`, `failing_policies_webhook`, `vulnerabilities webhook`).

This also means they need to be written when updating configs via POST `fleet/config`